### DISCUSSION TITLE
HA Core con Docker su Raspberry Pi ... primi passi ... caduto!

### DISCUSSION THREAD

**Discussion by xmxbx73** [indomus/forum#1005] [HUB: Home Assistant]
Ciao a tutti,
sono nuovo...e quindi inciampo parecchio.
Sto provando ad installare Home Assistant Core con Docker su Raspberry Pi.
fatto tutto (credo) e avviato HA, solo che non riesco ad accedervi da browser, nemmeno in locale.

Che verifiche potrei fare per capire dove ho sbagliato ?

Grazie per qualunque aiuto...
Marco.

**Comment by snakuzzo** [comment]
Ciao..
innanzi tutto ti chiederei di adeguare il titolo utilizzando parole chiave che potrebbero essere utili ad altri utenti che cercheranno qualcosa (es. Docker).

Poi...se parti con "credo" non andiamo bene :)
Fai vedere cos'hai fatto..come provi a creare il container

**Comment by xmxbx73** [comment]
Ho seguito la guida pubblicata sul sito.
per istanziare il container ho usato:
`docker-compose up -d`

se eseguo `docker ps` vedo che homeassistant è running.

**Reply by snakuzzo** [reply]
Per favore...non creare nuovi thread....rispondi sotto l'ultimo messaggio inviato

Se vai su 

`http://ip_del_server:8123` ... accedi ?

**Reply by calcioscacchi** [reply]
ciao, anche io ho avuto difficotà all'inizio quindi parlo per "esperienza fresca" :). 
Prima di tutto, assicurati che l'indirizzo IP che stai digitando sia quello con cui il raspberry è collegato alla tua rete.
Tuttavia, credo che stai incontrando anche tu difficoltà con la libreria libseccomp. Guarda la mia discussione a riguardo: aggiornata la libreria, sono riuscito ad accedere:

https://github.com/indomus/forum/discussions/784

**Reply by xmxbx73** [reply]
scusami...
no... non accedo.
ho provato sia con l'ip che con homeassistant.local che con il nome host.

**Reply by xmxbx73** [reply]
> ciao, anche io ho avuto difficotà all'inizio quindi parlo per "esperienza fresca" :). Prima di tutto, assicurati che l'indirizzo IP che stai digitando sia quello con cui il raspberry è collegato alla tua rete. Tuttavia, credo che stai incontrando anche tu difficoltà con la libreria libseccomp. Guarda la mia discussione a riguardo: aggiornata la libreria, sono riuscito ad accedere:
> 
> #784

era proprio quello. Grazie mille !!!!!!
